package.json

```diff
@@ -18,6 +18,9 @@
     "build/index.js",
     "build/example.js"
   ],
+  "dependencies": {
+      "prop-types": "^15.5.10"
```

Comment:
> Are we ready for this? Calypso isn't on React 15.5 yet

This particular version number is just for the new `prop-types` component, which while it mirrors some React versioning is a separate thing. According to its [migration guide](https://github.com/facebook/prop-types#compatibility), `prop-types` is compatible with 15.3.0 and up, so [the 15.4.0 in Calypso](https://github.com/Automattic/wp-calypso/blob/master/package.json#L106) should be fine.

It looks like it works with either 0.14.9 and higher or 15.3 and higher. So it seems at a minimum we'd want to bump our `react` dependency to 14.9. Given Calypso is already 15.4.0, maybe we just push it to 15.3.0? Added that in a separate commit and rebased to master.

> I would recommend moving this to a peer dependency

I don't really understand the alternatives too well, but the `prop-types` [migration guide](https://github.com/facebook/prop-types#how-to-depend-on-this-package) suggests leaving it in `dependencies`.